include/capstone/mips.h

```diff
@@ -684,7 +684,8 @@ typedef struct cs_mips_op {
 	mips_op_type type;	///< operand type
 	union {
 		mips_reg reg;		///< register id for REG operand
-		int64_t imm;		///< immediate value for IMM operand
+		int64_t imm;		///< signed immediate value for IMM operand
+		int64_t uimm;		///< unsigned immediate value for IMM operand
```

Comment:
`uint64_t uimm;`, right?
Also document for `is_unsigned` that `uimm` instead of `imm` is set if true.

Reply:
it's documented, i mean, it's obvious, no?